vtex-apps/slider
# Fix autoplay

#### What is the purpose of this pull request?

As the title says.

#### How should this be manually tested?

Check the [workspace](https://carousel--lesters.myvtex.com/) and see the slides transitioning.
Check [the master](https://lesters.myvtex.com/) to see stopping after the first slide transition

#### Screenshots or example usage

#### Types of changes
